src/applications/letters/sass/letters.scss

```diff
@@ -149,6 +149,7 @@
   }
 
   .form-expanding-group {
+    margin-bottom: 1.5em;
```

Comment:
I added this `margin-bottom` because the alert boxes were very close to the "Download letter" button when they appeared on the page. This layout didn't feel quite right and was easy to adjust given the local SCSS file.

---
![Screenshot 2024-11-22 155446](https://github.com/user-attachments/assets/474db3e2-3a98-4ef7-8dbb-4569665b9f07)